org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/impl/CompilerOptions.java

```diff
@@ -1640,7 +1644,7 @@ protected void resetDefaults() {
 		this.reportMissingJavadocCommentsOverriding = false;
 
 		// JSR bytecode inlining and sharing
-		this.inlineJsrBytecode = false;
+		this.inlineJsrBytecode = true;
```

Comment:
This could be deleted altogether, don't see a value.

Reply:
See the task above, I plan to delete that and the related constant in the next release to avoid breaking non-API changes now.